ush/rocoto/applications.py

```diff
@@ -187,7 +191,9 @@ def _cycled_configs(self):
             configs += ['awips']
 
         if self.do_wave:
-            configs += ['waveinit', 'waveprep', 'wavepostsbs', 'wavepostbndpnt', 'wavepostbndpntbll', 'wavepostpnt']
+            configs += ['waveinit', 'waveprep', 'wavepostsbs', 'wavepostpnt']
+            if self.do_wave_bnd:
```

Comment:
You are using self.do_wave_bnd before it is defined in the initializer

Reply:
Sorry.  Fixed now..